app/controllers/api/providers_controller.rb

```diff
@@ -85,6 +85,21 @@ def import_vm_resource(type, id = nil, data = {})
       end
     end
 
+    def options
+      if params[:ems_type].blank?
+        all_options = Hash[
+          ExtManagementSystem.descendants.collect do |ems|
+            [ems.ems_type, ems.respond_to?(:provider_settings) ? ems.provider_settings : {}]
```

Comment:
No, they come from "per-provider" options, for example: https://github.com/ManageIQ/manageiq-providers-kubernetes/pull/45